Manatee.Json/Schema/SchemaValidationContext.cs

```diff
@@ -77,42 +78,58 @@ public class SchemaValidationContext
 		/// Gets or sets whether the current validation run is for a meta-schema.
 		/// </summary>
 		public bool IsMetaSchemaValidation { get; set; }
+
 		/// <summary>
 		/// Miscellaneous data.  Useful for communicating results between keywords.
 		/// </summary>
 		/// <remarks>
 		/// Use <see cref="IJsonSchemaKeyword.ValidationSequence"/> to ensure that keywords are
 		/// processed in the correct order so that the communication occurs properly.
 		/// </remarks>
-		public Dictionary<string, object> Misc { get; } = new Dictionary<string, object>();
+		public Dictionary<string, object> Misc => _misc ??= new Dictionary<string, object>();
+
+		/// <summary>
+		/// Get or set if the validations for this context should track Evaluated Property Names.
+		/// </summary>
+		public bool ShouldTrackEvaluatedPropertyNames { get; }
```

Comment:
The change to the interface means that the name of this could probably change and be combined with the other as well.